var/spack/repos/builtin/packages/molden/package.py

```diff
@@ -29,6 +29,7 @@ class Molden(MakefilePackage):
     build_targets = ['clean', 'all']
 
     patch('for_aarch64.patch', when='target=aarch64:')
+    patch('ambor_gcc10.patch', when='%gcc@10:')
```

Comment:
How about doing something like to get rid of the patch?
```python
    def flag_handler(self, name, flags):
        if name == 'fflags':
            if self.spec.satisfies('%gcc@10:'):
                if flags is None:
                    flags = []
                flags.append('-fallow-argument-mismatch')

        return (flags, None, None)
```

Reply:
ya, makes sense. I will update the file. Thanks